internal/metrics/metrics.go

```diff
@@ -35,6 +40,31 @@ var (
 	once sync.Once
 )
 
+func getFromEnrichments(idx int) []string {
+	enrichements := os.Getenv(EnvMetricsEnrichment)
+	if len(enrichements) == 0 {
+		return nil
+	}
+	enrichmentSplit := strings.Split(enrichements, ";")
+	labels := make([]string, 0, len(enrichmentSplit))
+	for _, l := range enrichmentSplit {
+		kv := strings.Split(l, ":")
+		if len(kv) != 2 {
+			panic(errors.New("metrics enrichment expected format: label:value[;label:value]*"))
```

Comment:
I would rather this errors out, instead of a panic. Additionally, I would love a test for this